src/NServiceBus.Core/MessageDeserializationException.cs

```diff
@@ -32,8 +32,10 @@ public MessageDeserializationException(string messageId, Exception innerExceptio
         /// <summary>
         /// <see cref="SerializationException(SerializationInfo, StreamingContext)" />.
         /// </summary>
+        #pragma warning disable PC001
```

Comment:
https://github.com/dotnet/coreclr/blob/master/src/mscorlib/shared/System/Runtime/Serialization/SerializationException.cs#L35 should be supported right?

Reply:
I see you filed an issue about this being a false positive, which is what I was going to suggest.